<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgPrimitives</class>
 <widget class="QDialog" name="CQDlgPrimitives">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Primitive Shapes</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>X-size [m]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqXSize">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqYSize">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Y-size [m]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Z-size [m]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqZSize">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Y-subdivisions</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqYSubdiv">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqXSubdiv">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Z-subdivisions</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>X-subdivisions</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqZSubdiv">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Sides</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqSides">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Faces</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqFaces">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Disc subdivisions</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqDiscSubdiv">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="qqPure">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>161</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Create pure shape</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="qqOpen">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>161</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Open ends</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="qqCone">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>161</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Cone</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="qqSmooth">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>161</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Smooth shaded</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="qqOkCancel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>220</y>
     <width>351</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Material density [kg/m^3]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqDensity">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="qqNegativeVolume">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>161</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Shape has negative volume</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqNegativeVolumeScaling">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Inside scaling factor</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="qqDynamic">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>281</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Create dynamic and respondable shape</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqXSize</tabstop>
  <tabstop>qqYSize</tabstop>
  <tabstop>qqZSize</tabstop>
  <tabstop>qqXSubdiv</tabstop>
  <tabstop>qqYSubdiv</tabstop>
  <tabstop>qqZSubdiv</tabstop>
  <tabstop>qqSides</tabstop>
  <tabstop>qqFaces</tabstop>
  <tabstop>qqDiscSubdiv</tabstop>
  <tabstop>qqSmooth</tabstop>
  <tabstop>qqOpen</tabstop>
  <tabstop>qqCone</tabstop>
  <tabstop>qqDynamic</tabstop>
  <tabstop>qqPure</tabstop>
  <tabstop>qqDensity</tabstop>
  <tabstop>qqNegativeVolume</tabstop>
  <tabstop>qqNegativeVolumeScaling</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
